### PR TITLE
feat/document download

### PR DESCRIPTION
**Pull request by alexworker23**
- Send documents with organization data when registering an Organization
- Show attached documents in Organization view for OrganizationAdmin and Platform Admin
- Download documents 
orgViewNoIrec
orgViewWithIRec
severalDocs

### CODE REVIEW

**Comment by kosecki123** [comment]
Are these items labeled always as proof and signatory id?

Do we support multiple files for each? And how then this is going to be presented?

**Comment by alexworker23** [comment]
These items always labeled as Company Proof and Signatory ID on the Organization View, and these names are inherited when you download them.

Yes, multiple files are supported for each row. If there are multiple files for one document type they will appear one after another with indexes, just like Company Proofs on the last picture.

File: packages/origin-ui-core/src/components/Organization/DownloadDocuments.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    const link = document.createElement('a');
+                    link.href = url;
+                    link.setAttribute('download', name);
+                    link.style.display = 'none';
+                    document.body.appendChild(link);
+                    link.click();
+                    document.body.removeChild(link);
+                    setTimeout(() => {
+                        window.URL.revokeObjectURL(url);
+                    }, 100);
```
Comment (kosecki123) [inline packages/origin-ui-core/src/components/Organization/DownloadDocuments.tsx]: I think there is no reason to use the download via XHR call. Just use `getLinks` to generate URL and render them on the document, browser will handle the download.

**Review by kosecki123 — APPROVED** [review]

File: packages/origin-ui-core/src/components/Organization/DownloadDocuments.tsx (hunk 1 above)
Comment (kosecki123) [reply packages/origin-ui-core/src/components/Organization/DownloadDocuments.tsx]: This flow is currently impossible as <a> element links are not passing the custom headers due to security concerns.

**Pull request merged**